docs/Server.md

```diff
@@ -262,6 +262,26 @@ const fastify = require('fastify')({
 })
 ```
 
+<a name="factory-modify-core-objects"></a>
+### `modifyCoreObjects`
+
+By enabling the `modifyCoreObjects` option, Fastify will set `ip`, `ips`, and `hostname` to Node's request object. And set `log` to Node's request and response objects for logging. The consequence of enabling this option is these properties will be available in the `raw` object of Fastify's Request. For example, `ip`, `ips`, and `hostname` values can be accessed from raw `request`.
+
+```js
+const fastify = Fastify({ modifyCoreObjects: true })
+
+fastify.get('/', (request, reply) => {
+  console.log(request.raw.ip)
+  console.log(request.raw.ips)
+  console.log(request.raw.hostname)
+})
+```
+
++ Default: `false`
```

Comment:
@mcollina For backwards compatibility, shouldn't this option default to `true`?

Reply:
Yes actually! Would you have time to update this?

Reply:
Changed in 9b6b23f

Reply:
@shwei I'm not sure where that commit ended up, but I incorporated your changes in #1517.

Reply:
Thanks, @nwoltman! By the way, how would this new option get backported?? (Very eager here!)

Reply:
The master branch is currently still the main branch for Fastify v2, so this is already on track to be released in the next minor version of Fastify 👍